src/main/java/org/glyptodon/guacamole/Stress.java

```diff
@@ -227,14 +247,24 @@ else if (instruction.getOpcode().equals("error")) {
 
                 // Otherwise, add total bytes
                 else {
-                    for (String arg : instruction.getArgs())
+                    for (String arg : instruction.getArgs()) {
+                        metrics.addBytes(arg.length());
                         bytes += arg.length();
+                    }
+
+                    // Add frames to metrics counter
+                    if (instruction.getOpcode().equals("png") ||
```

Comment:
Right.. I fixed that up to keep track of both true frames and draw operations (which was what I really was after originally).